packs/feat-effects/effect-resounding-bravery.json

```
@@ -4,7 +4,7 @@
     "name": "Effect: Resounding Bravery",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Resounding Bravery]</p>\n<p>You gain a +1 status bonus to saves and a number of temporary Hit Points equal to half your level. On a Critical Success, you gain a +2 status bonus to saves and a number of temporary Hit Points equal to your level.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Resounding Bravery]</p>\n<p>You gain a +1 status bonus to saves and a number of temporary Hit Points equal to half your level. Or a +2 status bonus to saves and a number of temporary Hit Points equal to your level if it was against a fear effect.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Resounding Bravery]</p>\n<p>You gain a +1 status bonus to saves and a number of temporary Hit Points equal to half your level. These benefits are doubled if you critically succeed against a fear effect.</p>"
```
Personally I think the wording of the feat is a bit nicer looking :)